src/utils/ComponentUtils.ts

```diff
@@ -15,3 +16,19 @@ export const DisplayClass = {
 };
 
 export type ComponentContent = (new () => React.Component) | string | (() => JSX.Element);
+
+export const getBasicDocumentLink = (url: string, title: string = ''): ILinkSvgProps => {
```

Comment:
Is this used?

Reply:
Yep, its only an util to create my link faster

Reply:
this will surely be useful 👍 thanks